Clarify protocols and formats trade-off

bgever opened the pull request:
The original statement seems to be the opposite of what was meant to be said.

If the protocols and formats are not openly available, developers will find it harder to understand how to use them.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

bgever committed:
Clarify protocols and formats trade-off 

If the protocols and formats are not openly available, developers will find it harder to understand how to use them.
